examples/Polls Hypermedia API.md

```diff
@@ -0,0 +1,408 @@
+FORMAT: 1A
+HOST: http://polls.apiblueprint.org/
+
+# Polls
+
+Polls is a simple web service that allows consumers to view polls and vote in them. You can view this documentation over at [Apiary](http://docs.pollshypermedia.apiary.io/).
+
+# Polls API Root [/]
+
+This resource does not have any attributes. Instead it offers the initial API affordances.
+
+## Retrieve the Entry Point [GET]
+
++ Response 200 (application/vnd.siren+json)
+
+        {
+            "links": [
+                {
+                    "rel": [ "questions" ],
+                    "href": "\/questions"
+                }
+            ]
+        }
+
++ Response 200 (application/hal+json)
```

Comment:
Is it required to explicitly put this? I though it would be a given that [content negotiation](http://www.w3.org/Protocols/rfc2616/rfc2616-sec12.html) as per HTTP 1.1 specification would work to get the expected result.

(I am happy to add it if you see this is needed).